data/panel.html

```diff
@@ -58,5 +101,25 @@
   </div>
   <hr>
   <p class="note">On E10S builds, Statuser will <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1228437" target="_blank">only measure main thread hangs</a>.</p>
+  <hr>
+  <h1>MOST RECENT HANGS</h1>
+  <div id="hangStacks">
+    <!-- hang stack entries are added here -->
+    <div>
```

Comment:
You probably don't want to leave this fake UI in, just in case something explodes and lets the user see this.